src/main/java/com/google/sps/servlets/CasesDataServlet.java

```diff
@@ -76,25 +80,100 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(reportsJson);
   }
 
-  /**
-   * Returns Json string of all reports
-   */
   public String getReportsJson() {
     return reportsJson;
   }
 
   /**
-   * Represents active number of cases at a specific lat lng point
+   * Establish connection to live Coivd-19 data set
+   */
+  private Scanner connectToData() {
+    URL covidDataUrl = null;
+    HttpURLConnection connection = null;
+    int responseCode = 0;
```

Comment:
more of line 93 should happen throughout the code. There are some "magic" numbers used for the indexes of cells in lines 58-62, i'm sure they're right but I have no idea what you're doing with them.